api/v1/routes/user.py

```diff
@@ -60,3 +60,19 @@ async def reactivate_account(request: Request, db: Session = Depends(get_db)):
         status_code=200,
         message='User reactivation successful',
     )
+
+
+@user.patch("/current-user/", status_code=status.HTTP_204_NO_CONTENT)
```

Comment:
should be `@user.delete`  not patch

Reply:
Noted boss.

I saw patch in the spreadsheet that's why I put it there.